Tests/ValkeyTests/Utils/TestTracer.swift

```diff
@@ -0,0 +1,209 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the valkey-swift open source project
+//
+// Copyright (c) 2025 the valkey-swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of valkey-swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Tracing open source project
+//
+// Copyright (c) 2020-2023 Apple Inc. and the Swift Distributed Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Distributed Tracing project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if DistributedTracingSupport
+import Dispatch
+import Foundation
+import Instrumentation
+import ServiceContextModule
+import Tracing
+
+/// Only intended to be used in single-threaded testing.
+final class TestTracer {
```

Comment:
I'm pretty sure this should live in DistributedTracing in a separate module called `TestUtils`. Also this should be proper Sendable from the beginning. Adding annotations to use `Mutex` is fine.

Reply:
Yes that'd be fine

Reply:
Good idea, I'll look into that too.